src/analysis/retail/priest/discipline/modules/spells/PowerWordRadiance.tsx

```diff
@@ -0,0 +1,78 @@
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent, BeginCastEvent, ApplyBuffEvent } from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+import Atonement from './Atonement';
+
+type RadianceInfo = {
+  cast: CastEvent;
+  goodCast: boolean;
+  onAtoned: boolean;
+};
+
+class PowerWordRadiance extends Analyzer {
+  radiancesOnAtonedTarget = 0;
+  atonementsPreRadiance = 0;
+  lastPWR: CastEvent | null = null;
```

Comment:
The above three properties seem to be unused. If you need to access them for the guide i'd suggest you store them together with the other data in `RadianceInfo`.